lib/powerline.sh

```diff
@@ -2,7 +2,9 @@
 
 print_powerline() {
 	local side="$1"
```

Comment:
Is shellcheck not complaining about `side` as well?

Reply:
not here, but it was on ln:150 and 152 of the current changes :)